### DISCUSSION TITLE
文件夹太长

### DISCUSSION THREAD

**Discussion by Free-Aaron-Li** [linuxdeepin/developer-center#2995] [General | 常规讨论]
文件夹长度太长了，建议像windows11只保留几个比较重要的按键，其他按键集成到一个按键上，要的时候在打开

**Comment by JiaOkay** [comment]
描述不够清楚 最好是附上图片

**Comment by zccrs** [comment]
这个着实没看懂是啥意思

**Comment by myml** [comment]
是觉得文件管理器中文件夹右键菜单太多了吗？

**Comment by BLumia** [comment]
确实没看懂是什么意思，如果是指面包屑上显示的路径的话，我们目前和 windows 很类似，前半部分能省略的也会省略（比如照片目录的面包屑不会显示成 home > username > Pictures 而是直接 Pictures ...

实在看不懂在问什么，我就先挪到讨论版去了...
